contracts/courtroom_test.go

```diff
@@ -194,36 +201,43 @@ func openClaimForReflectorGame(t *testing.T, clientContent string, serviceConten
 	if err != nil {
 		t.Fatalf("balanceOfClientBefore : expected no error, got %v", err)
 	}
-	if clientContent != serviceContent {
-		if amountStakedBefore.Int64()-amountStakedAfter.Int64() != 5 {
-			t.Fatalf("After a valid claim proccess : amountStaked at the contract should reduce by 5  ( got %d)", (amountStakedBefore.Int64() - amountStakedAfter.Int64()))
+	if (clientContent != serviceContent) && (numberOfBlocksToWait <= PromiseTillNextBlocks) {
```

Comment:
why ? this wait logic should be in the contract (i know it is not easy) but at least comment